wandb/apis/public/runs.py

```diff
@@ -144,26 +230,32 @@ def __init__(
 
     @property
     def length(self):
+        """Returns the total number of runs."""
         if self.last_response:
             return self.last_response["project"]["runCount"]
         else:
             return None
 
     @property
     def more(self):
+        """Returns `True` if there are more runs to fetch. Returns
+        `False` if there are no more runs to fetch.
+        """
```

Comment:
Same comment here about annotating the return type and shortening the summary line, if possible.